demos/common/cpp/WORKSPACE

```diff
@@ -23,14 +23,30 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 git_repository(
     name = "tensorflow_serving",
     remote = "https://github.com/tensorflow/serving.git",
-    tag = "2.6.5"
+    tag = "2.6.5",
+    patch_args = ["-p1"],
+    patches = ["net_http.patch", "listen.patch"]
```

Comment:
Do you know why we did not need those patches ealier? We do not use HTTP server from TFS in this demo